docs/docsite/rst/reference_appendices/galaxy.rst

```diff
@@ -99,7 +99,7 @@ Each role in the file will have one or more of the following attributes:
    scm
      Specify the SCM. As of this writing only *git* or *hg* are supported. See the examples below. Defaults to *git*.
    version:
-     The version of the role to download. Provide a release tag value, commit hash, or branch name. Defaults to *master*.
+     The version of the role to download. Provide a release tag value, commit hash, or branch name. Defaults to the branch set as a default in the repository otherwise defaults to the *master*.
```

Comment:
```suggestion
     The version of the role to download. Provide a release tag value, commit hash, or branch name. Defaults to the branch set as a default in the repository, otherwise defaults to the *master*.
```